config/manager/manager.yaml

```diff
@@ -37,6 +37,26 @@ spec:
                 fieldPath: metadata.namespace
           - name: SECRET_NAME
             value: kserve-webhook-server-cert
+        livenessProbe:
+          failureThreshold: 5
+          initialDelaySeconds: 10
+          httpGet:
+            path: /healthz
+            port: 8081
+          timeoutSeconds: 5
+        readinessProbe:
+          initialDelaySeconds: 10
+          periodSeconds: 5
+          httpGet:
+            path: /readyz
+            port: 8081
+          timeoutSeconds: 5
+        startupProbe:
```

Comment:
Do we need startupProbe the webhook container should start pretty quick, here failureThreshold*periodSecond=500s which is almost 8min.

Reply:
Removed the startup probe. Increased readiness failureThreshold to 10. (failureThreshold*periodSecond)+initialdelay=60s. Sounds good ?